src/other/broadcasting.jl

```diff
@@ -33,6 +33,13 @@ function copyto_widen!(res::AbstractVector{T},
     return res
 end
 
+function getcolbc(bcf::Base.Broadcast.Broadcasted{Style}, colind) where {Style}
+    # we assume that bcf is already flattened and unaliased
+    Base.Broadcast.Broadcasted{Style}(bcf.f,
+        map(x->Base.Broadcast.extrude(x isa AbstractDataFrame ? x[colind] : x),
```

Comment:
Move this to a separate line (with an intermediate variable) for readability?

Reply:
done